app/models/checkout.rb

```diff
@@ -10,4 +11,11 @@ class Checkout < ApplicationRecord
   def due_date
     self.created_at + CHECKOUT_PERIOD_IN_DAYS.days
   end
+
+  def fine
+    days_overdue = (Time.zone.now - self.due_date).to_i/1.day
+    if days_overdue > 0
```

Comment:
Hi @derekprior, I originally refrained from returning 0 because I did not want the view cluttered with $0.00 for books that had no fine. However, that/you raise(s) a good point that the logic to display how the information looks should live in the view, and not be the concern of the model methods. I now return 0 if the book is not overdue.